bryant-bryantbuilt/week-6/my-react-todo/src/containers/TodosContainer.js

```diff
@@ -0,0 +1,71 @@
+import React, {Component} from 'react'
+import TodoModel from '../models/Todo'
+import Todos from '../components/Todos'
+import CreateTodoForm from '../components/CreateTodoForm';
+
+class TodosContainer extends Component {
+  state = {
+      todos: []
+  }
+
+  componentDidMount(){
+    this.fetchData()
+  }
+
+  fetchData(){
+    TodoModel.all().then( (res) => {
+      this.setState ({
+        todos: res.data.todos,
+        todo: ''
+      })
+    })
+  }
+
```

Comment:
Overall your container functions are looking good. Since now you know what was going on with the update function after brocks lecture you can revisit and correct this issue.